src/ui/resource/Image.js

```diff
@@ -147,7 +147,7 @@ exports = Class(lib.PubSub, function () {
     // create an image if we don't have one
     if (!img) {
       img = new Image();
-      img.crossOrigin = 'use-credentials';
```

Comment:
I don't we should change it like this. Mobile might break.

Reply:
Checked on ios with no issues.

Reply:
what is the purpose of use-credentials ?

Reply:
For user-authentication using CORS.